Robustness Test Implementation

dkosteck opened the pull request:
Not sure if layout is the desired base, but since it is the most up to date, that is where this pull request is targeted. 

This branch is the Implementation of SR-IOV.MultipleVFCreation.withMTU:
- Allow longer timeouts (with default 5 seconds) in execute
- Implement the test in the SR-IOV.MultipleVFCreation.withMTU directory
- Add reused fixtures to conftest

Note that the directory structure and file naming convention may not be as desired, this is something we were still discussing.

jianzzha commented on sriov/tests/conftest.py:
```diff
@@ -15,4 +15,28 @@ def dut(settings):
+
+@pytest.fixture
+def reset_vfs(dut, settings, pf_net_path):
+    yield
```
maybe we can have a single clean up function which cover both the pre-test initialization and post-test clean up, something like:
@pytest.fixture(autouse=True)
def common_cleanup(dut, settings, pf_net_path):
     reset_command(dut, settings, pf_net_path)
     yield
     du.stop_testpmd()
      reset_command(dut, settings, pf_net_path)

dkosteck commented:
See the most recent push to robustness for updates, including the following:
- Removed dots and dashes from directory as well as test name, replaced with underscores
- Condensed cleanup command in conftest
- Added sample extras command to conftest (NOTE: Currently not used for identifiers)
- Moved pf fixtures to testdata fixture
- Added logic for looping through pfs for robustness tests

A note on the extras command: I know we want to put something like "SR-IOV.MultipleVFCreation.withMTU" in the report when running "test_SR_IOV_MultipleVFCreation_withMTU.py", however we need some kind of mapping for this to happen. We should have a discussion about this (as we need some kind of mapping to accomplish this)
